.gitignore

```diff
@@ -12,12 +12,6 @@ lcov.info
 .idea
 
 /*.prql
-_*.prql
-**/project/**/*.prql
-/*.rq
-_*.rq
-/*.pl
-_*.pl
```

Comment:
Not sure why these are here; looks like I added them. Using `foo.rq.yaml` makes more sense